### DISCUSSION TITLE
await chat(), The Markdown text presented is chaotic

### DISCUSSION THREAD

**Discussion by alkene5** [johnlindquist/kit#1299] [Q&A] [answered]
<img width="768" alt="image" src="https://github.com/johnlindquist/kit/assets/61105068/f955d520-ca60-4b2a-8d78-e65cedb01900">

**Comment by johnlindquist** [comment, marked as answer]
@alkene0005 I'm not able to reproduce from the "ChatGPT" demo.

<img width="811" alt="CleanShot 2023-07-05 at 15 55 39@2x" src="https://github.com/johnlindquist/kit/assets/36073/cfc56d9e-74ad-4456-a264-6bba84fbf228">

You do need to make use the `md` helper to format any markdown codeblocks into HTML before setting the message:

```js
  let htmlMessage = md(currentMessage)
  chat.setMessage(-1, htmlMessage)
```

**Reply by alkene5** [reply]
I forgot to add md()  😂 thank you
